app/models/partner.rb

```diff
@@ -26,8 +27,11 @@ class Partner < ApplicationRecord
   enum status: { uninvited: 0, invited: 1, awaiting_review: 2, approved: 3, error: 4, recertification_required: 5, deactivated: 6 }
 
   belongs_to :organization
-  has_many :distributions, dependent: :destroy
+  belongs_to :partner_group, optional: true
```

Comment:
does this mean a `Partner` can only belong to at most one `PartnerGroup`?

Reply:
That is correct. The thought behind this is that keeping to a single association might be good enough for today's needs. If we discover it is lacking by complaints, then I think we can update it to include multiple associations.

Reply:
sounds good